src/test/resources/consumer.go

```diff
@@ -0,0 +1,132 @@
+package borges
```

Comment:
As discussed in https://github.com/src-d/gemini/pull/97#issuecomment-381856684 - in my experience, it's safer to be conservative and develop a habit of adding a comment with original license and, if possible, an origin of source code files, if copied from other projects.

Reply:
than it wouldn't be identical